docs/samples/browser-plugin-meetings/app.js

```diff
@@ -292,6 +292,8 @@ const meetingsListMsgElm = document.querySelector('#meetings-list-msg');
 const meetingsListElm = document.querySelector('#meetings-list');
 const meetingsAddMediaElm = document.querySelector('#meetings-add-media');
 const meetingsLeaveElm = document.querySelector('#meetings-leave');
+const meetingStartLinkElm = document.querySelector('#meetings-start-link');
```

Comment:
Had to add some new elements to sample app - users can choose which one they want to use.